crates/bitwarden/src/crypto/master_key.rs

```diff
@@ -52,6 +52,14 @@ impl MasterKey {
         let dec: Vec<u8> = user_key.decrypt_with_key(&stretched_key)?;
         SymmetricCryptoKey::try_from(dec.as_slice())
     }
+
+    pub(crate) fn encrypt_user_key(&self, user_key: &SymmetricCryptoKey) -> Result<EncString> {
+        use super::KeyEncryptable;
+
+        let stretched_key = stretch_master_key(self)?;
+
+        (user_key.to_vec().as_slice()).encrypt_with_key(&stretched_key)
```

Comment:
I previously used the `EncString::encrypt_aes256_hmac` since we want to be certain it's encrypted that way.